src/Main.java

```diff
@@ -0,0 +1,31 @@
+public class Main {
+    public static void main(String[] args) {
+        FirePokemon firePokemon = new FirePokemon("Charmander", "grass", 200, 5);
+        WaterPokemon waterPokemon = new WaterPokemon("Empoleon", "plankton", "beep beep", 6);
+        GrassPokemon grassPokemon = new GrassPokemon("Oddish", "seeds", 0.5, 5);
+        ElectricPokemon electricPokemon = new ElectricPokemon("Pikachu", "leafs", 4, 35);
+        firePokemon.introduce();
+        firePokemon.printAttack();
+        firePokemon.solarBeam();
+        firePokemon.thunderPunch();
+        System.out.println(" ");
```

Comment:
Waarom print je hier een spatie? (" ")
Is het om een witregel te printen? In dat geval kun je gewoon println aanroepen zonder een " " als argument te geven. Zonder argument wordt er een witregel geprint.